tdrs-frontend/src/components/EditProfile/EditProfile.test.js

```diff
@@ -5,10 +5,18 @@ import { Provider } from 'react-redux'
 import configureStore from 'redux-mock-store'
 import { fireEvent, render } from '@testing-library/react'
 
+import { MemoryRouter, Redirect } from 'react-router-dom'
 import EditProfile, { validation } from './EditProfile'
 
 describe('EditProfile', () => {
-  const initialState = { stts: { stts: [], loading: false } }
+  const initialState = {
+    auth: { authenticated: true, user: { email: 'hi@bye.com' } },
+    stts: { sttList: [], loading: false },
+    requestAccess: {
+      requestAccess: false,
+      loading: false,
+    },
```

Comment:
This just adds the requestAccess module to the test store to mirror what was added in this PR.